### ISSUE TITLE
Touchpad

### ISSUE THREAD

**Issue by Utkub24** [infernix/samsung_tabpro_s#8]
It's a relief to finally see others having the same problems with the Tab Pro S.
Any updates on the touchpad problem? 
The touchpad not working properly is my biggest concern at the moment, if I can't fix it I might return to Windows.

**Comment by infernix** [comment]
I havent tried this with recent distros/kernels in a very long time. So no idea. What distro have you tried?

**Comment by Utkub24** [comment]
EndeavorOS KDE. It was virtually unusable with Windows

**Comment by infernix** [comment]
That seems recent enough. I can give it another try but not before end of september. Does sound work though?

**Comment by Utkub24** [comment]
Nope, external headphones work just fine but the internal speakers don't work.

**Comment by Nprocessclan** [comment]
Sometimes my touchpad does not working, but suddenly I realised that it starts working if I press CAPS LOCK. Its like the On/Off button.

Hope it helps someone
